core/config_api.php

```diff
@@ -154,19 +148,18 @@ function config_get( $p_option, $p_default = null, $p_user = null, $p_project =
 
 					switch( $t_type ) {
 						case CONFIG_TYPE_FLOAT:
-							$t_value = (float)$t_raw_value;
+							return (float)$t_raw_value;
 							break;
 						case CONFIG_TYPE_INT:
-							$t_value = (int)$t_raw_value;
+							return (int)$t_raw_value;
 							break;
 						case CONFIG_TYPE_COMPLEX:
-							$t_value = json_decode( $t_raw_value, true );
+							return json_decode( $t_raw_value, true );
 							break;
```

Comment:
Remove the `break` in the above 3 cases - they are dead code due to the use of `return` statement.

Reply:
Done

Changed also function config_get_access to the same logic like function config_is_set (return instead of break 2)